### DISCUSSION TITLE
[swager] Add docExpansion=none to collapse all entries

### DISCUSSION THREAD

**Discussion by uri200** [magma/magma#5581] [Ideas]
Hey @AndyLKhuu , good job with the new swagger interface.

Would it be possible to add to your side links the option `docExpansion=none` so all the endpoints show collapsed?
Or maybe even better, add a button to switch back and forth?

See the example below

https://api-staging.magma.etagecom.io/swagger/v1/ui/?docExpansion=none

![image](https://user-images.githubusercontent.com/16157139/111557131-4f6f7900-8749-11eb-9f36-e9a46b92a7b5.png)

**Comment by AndyLKhuu** [comment]
@uri200 Hey Oriol, thanks for bringing this up. I actually already have a PR out to do this. https://github.com/magma/magma/pull/5538. It's pending review so if you want to take a look,  I think this solves what you were hoping for.

**Reply by uri200** [reply]
awesome!

**Reply by hcgatewood** [reply]
@AndyLKhuu love it when a plan comes together 🙂
